config/webpack/webpack.staging.js

```diff
@@ -0,0 +1,25 @@
+const path = require('path');
+const webpack = require('webpack');
+const {merge} = require('webpack-merge');
+const {BundleAnalyzerPlugin} = require('webpack-bundle-analyzer');
+const dotenv = require('dotenv');
+const common = require('./webpack.common.js');
+
+const env = dotenv.config({path: path.resolve(__dirname, '../../.env.staging')}).parsed;
```

Comment:
Is this the only difference between prod/staging? Is there a way to DRY this up more?

Reply:
DRYed up a bit 👍